bagitobjecttransfer/recordtransfer/models.py

```diff
@@ -88,6 +88,7 @@ def __str__(self) -> str:
     status = models.CharField(
         max_length=2, choices=SessionStatus.choices, default=SessionStatus.CREATED
     )
+    user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
 
     @classmethod
     def new_session(cls) -> UploadSession:
```

Comment:
Could you make it so that `new_session` can accept a user object when the session is created?

```suggestion
    def new_session(cls, user: Optional[User] = None) -> UploadSession:
```

Reply:
I like this idea - it's certainly neater than setting the user after the session is created.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/321/commits/54c10848d8030c940eddf4567d37a6de2a10d35e